subliminal/providers/subscenter.py

```diff
@@ -191,6 +192,7 @@ def query(self, title, season=None, episode=None):
                         subtitle_key = subtitle_item['key']
                         downloaded = subtitle_item['downloaded']
                         release = subtitle_item['subtitle_version']
+                        h_version = subtitle_item['h_version']
```

Comment:
Well, if you can make it so the test is accurate with regards to the current version that'd be great.